lua/oil/columns.lua

```diff
@@ -205,12 +205,12 @@ if has_devicons then
         end
       end
       if type == "directory" then
-        return { " ", "OilDir" }
+        return { conf.directory, "OilDir" }
       else
         local icon
         local hl
         icon, hl = devicons.get_icon(name)
-        icon = icon or ""
+        icon = icon or conf.default_file
```

Comment:
These will fail for anyone who specifies "icon" in the config with no options. Can you add a fallback to the original value? You will need to handle the case where conf is nil

Reply:
Okay